lib/message_queue.cpp

```diff
@@ -68,7 +68,7 @@ MessageHandler::MessageHandler() : running(true) {
             this->message_queue.pop();
             lock.unlock();
 
-            auto data = *message.get();
+            auto data = *message.data;
```

Comment:
This is expensive, as the whole `json` object will get copied.  Either do `const auto&` or use `data->at(..)` in the following.

Reply:
reference now used